awx/main/conf.py

```diff
@@ -43,6 +43,16 @@
     category_slug='system',
 )
 
+register(
+    'MANAGE_ORGANIZATION_AUTH',
+    field_class=fields.BooleanField,
+    label=_('Organizations Can Manage Users and Teams'),
+    help_text=_('Controls whether Orgainzations have the privileges to create and manage users and teams. '
```

Comment:
I feel like you might want to say "Organization admins can manually..."

or something like that, because linguistically, someone might say that the "organization" is managing teams _via_ LDAP or whatever